scripts/ci/publish.ts

```diff
@@ -1003,12 +1008,16 @@ async function publishPackages(
         })
       }
 
-      if (process.env.BUILDKITE) {
-        await run(pkgDir, `pnpm run build`, dryRun)
-      }
-
```

Comment:
We have `"prepublishOnly": "pnpm run build"` for that, no need to do it here again